tests/containers/podman_pods.pm

```diff
@@ -18,6 +18,43 @@ use serial_terminal 'select_serial_terminal';
 use version_utils qw(is_sle is_opensuse is_sle_micro);
 use containers::k8s qw(install_k3s uninstall_k3s);
 
+sub check_container_nspid {
+    # expect set hostpid
+    my $hostpid = shift;
+    my $config = 'hello-kubic.yaml';
+    my @errors = ();
+
+    record_info('NSPid', sprintf('Expect HostPid == %s', !!$hostpid ? 'True' : 'False'));
```

Comment:
Is this line correct?

Reply:
wdym?

Reply:
What does `!!$hostpid` mean?

Reply:
perl idiom to convert anything to boolean.
https://www.perlmonks.org/?node_id=374287;replies=1;displaytype=print